forge/csrc/forge_passes.cpp

```diff
@@ -25,7 +25,7 @@
 #include "passes/generate_initial_flops_estimate.hpp"
 #include "passes/hoist_transforms_to_inputs.hpp"
 #include "passes/insert_inverse_on_io.hpp"
-#include "passes/limit_to_4d_reshape.hpp"
+#include "passes/limit_to_5d_reshape.hpp"
```

Comment:
Can you create issues to deprecate these limitations? More precisely, to mark it with "exploration" label, and see why we need to impose these limitations further :))

Reply:
Created issue to explore more on this issue - https://github.com/tenstorrent/tt-forge-fe/issues/679